tests/ci/android/AWSLCAndroidTestRunner/app/src/main/cpp/CMakeLists.txt

```
@@ -0,0 +1,38 @@
+# For more information about using CMake with Android Studio, read the
+# documentation: https://d.android.com/studio/projects/add-native-code.html
+
+# Sets the minimum version of CMake required to build the native library.
+
+cmake_minimum_required(VERSION 3.4.1)
+
+# AWS lib
+set(path_to_awslc "${CMAKE_CURRENT_LIST_DIR}/../../../../../../../../")
```

```suggestion
set(path_to_awslc "${CMAKE_SOURCE_DIR}")
```
CMAKE_SOURCE_DIR is the path to the top level project where `project(AWSLC NONE)` is.

---

When using `add_subdirectory` to add as a CMake dependency, `CMAKE_SOURCE_DIR` points to the original source, which is this specific file in this case.